stubs/dom.stub

```diff
@@ -155,11 +155,37 @@ class DOMProcessingInstruction
 }
 
 /**
+ * @template-covariant TNode as DOMNode
+ * @implements Traversable<int, TNode>
+ * @implements IteratorAggregate<int, TNode>
+ *
  * @property-read int $length
  */
-class DOMNamedNodeMap
+class DOMNamedNodeMap implements Traversable, IteratorAggregate, Countable
```

Comment:
It shouldn't implements both Traversable and IteratorAggregate.

See
https://github.com/JetBrains/phpstorm-stubs/blob/6c034ae018ed7ac2cacf2474f824aede73658afa/dom/dom_c.php#L1381
F

Also I dunno if you need to redeclare the template (and all the methods) here or if the fix on the DOMNode is not enough.

Reply:
@VincentLanglet I just copied it from `DOMNodeList` at https://github.com/phpstan/phpstan-src/blob/c0e6f0e0c00aa1e7c8ffe395ffdf4f881985a6c9/stubs/dom.stub#L75. Do you want me to fix `DOMNodeList` as well?

Reply:
If it's done this way for DOMNodeList you can just ignore my remark I think

Reply:
Please fix both DOMNodeList and DOMNamedNodeMap here.

There are some classes where it'd make sense because of the logic here https://github.com/ondrejmirtes/BetterReflection/blob/61b25baaca1ea904447f46d975a4ae7c99b722e6/src/SourceLocator/SourceStubber/PhpStormStubsSourceStubber.php#L528-L574, but it's not true for any `DOM*` classes.

Reply:
I've now removed `Traversable` from all DOM classes.